Project2/style.css

```diff
@@ -0,0 +1,25 @@
+.card {
```

Comment:
No font declared on the body or .card, so this will display in whatever font is the default on the user's system or browser. That is a lot of leeway to provide as a designer, imo.